lib/wifi_user/use_case/sponsor_users.rb

```diff
@@ -98,11 +101,13 @@ def send_confirmation_singular(sponsor_address, sponsees)
     )
   end
 
-  def send_sponsor_failed(sponsor_address)
+  def send_sponsor_failed(sponsor_address, failed_sponsees: [])
```

Comment:
I would call this `send_failed_sponsoring_email`, or something like that. 

Think there's the opportunity to make this more clear through naming, at least.